### PR TITLE
Aarch64 compilefixes

### PR DESCRIPTION
**Pull request by sknsean**

### CODE REVIEW

File: recipes/expat/expat.inc [hunk 1]
```diff
@@ -6,6 +6,9 @@ RECIPE_TYPES = "machine native sdk"
 
 inherit autotools make-vpath
 
+#Only used for Aarch64
+AUTOCONFDIR = "/conftools"
```
Comment (kimrhh) [inline recipes/expat/expat.inc]: are you sure it is not picked up in the other builds as well?

File: recipes/libiconv/libiconv.inc [hunk 2]
```diff
@@ -10,6 +10,9 @@ SRC_URI = "ftp://ftp.gnu.org/pub/gnu/libiconv/libiconv-${PV}.tar.gz"
 
 inherit library autotools
 
+#Only used for Aarch64
+AUTOCONFDIR = "/build-aux"
```
Comment (kimrhh) [inline recipes/libiconv/libiconv.inc]: are you sure it is not picked up in the other builds as well?

File: recipes/expat/expat.inc (hunk 1 above)
Comment (sknsean) [inline recipes/expat/expat.inc]: The function of the AUTOCONFDIR can be seen in the aarch64 PR in core :-)

File: recipes/libiconv/libiconv.inc (hunk 2 above)
Comment (sknsean) [inline recipes/libiconv/libiconv.inc]: yes

File: recipes/expat/expat.inc (hunk 1 above)
Comment (kimrhh) [inline recipes/expat/expat.inc]: ok, but you could just only set that var for that arch, I would much prefer that anyway

Comment (sknsean) [inline recipes/expat/expat.inc]: In core it's only read for aarch64 :-) So i guess i could add :TARGET_ARCH_aarch64, but is allready there in core... We wait for @esbens view of this before i do any more work on this

File: recipes/libiconv/libiconv.inc (hunk 2 above)
Comment (esben) [inline recipes/libiconv/libiconv.inc]: I believe that it is included in the other builds, just not used for anything.

**Comment by kimrhh** [comment]
is this replaced by one of your oe-lite/core PRs ?

**Comment by sknsean** [comment]
No, this is still needed to replace the config.{guess,sub} :-)

File: recipes/fuse/fuse_2.9.4.oe [hunk 3]
```diff
@@ -1,6 +1,7 @@
 require fuse.inc
 
-SRC_URI += "file://dont-run-updaterc.d-on-host.patch"
+SRC_URI += "file://dont-run-updaterc.d-on-host.patch \
+            file://fuse_kernel.h_clean_includes.patch"
```
Comment (esben) [inline recipes/fuse/fuse_2.9.4.oe]: There is also fuse 2.9.5.  Does that also require this patch?

Comment (sknsean) [inline recipes/fuse/fuse_2.9.4.oe]: Maybe. It's seems the upstream is in a messy state...
https://github.com/libfuse/libfuse the patch is not applied
http://git.baserock.org/cgit/delta/fuse.git/ the patch is applied

I do not have time to upgrade fuse.

**Pull request merged**